javascript-source/systems/auctionSystem.js

```diff
@@ -19,14 +19,30 @@
     var auction = {
             increments: 0,
             minimum: 0,
-            topUser: 0,
+            topUser: 'None',
             topPoints: 0,
             timer: 0,
-            status: false
+            isActive: false,
+            isEnding: false,
+            usePoints: true
         },
         a,
-        b;
+        b,
+        c,
+        //endTimer,
+        extTime = $.getSetIniDbNumber('auctionSettings', 'extTime', 15),
+        warnTime = 10;
+
 
+    /**
+     * @function usePoints
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
   * @function usePoints
```